polynomial.c

```diff
@@ -1,128 +1,157 @@
-#include<stdio.h>
-#include<stdlib.h>
-#include<mpi.h>
+/*
+  COSC 6060
+  Matthew Dupont and Louis Finney
+  Assignment 2 - MPI work distribution
+*/
 
-#define MAX 50000
-#define COEFFICIENT 1
+#include <stdio.h>
+#include <stdlib.h>
 
+#include <mpi.h>
 
-double power(double x, int degree)
-{     
```

Comment:
ugh. looks like the diff on this guy got really mangled. Might be better to just view as it is...